### PR TITLE
Seed_all in resnet example

### PR DESCRIPTION
**Pull request by Landanjs**

### CODE REVIEW

**Review by coryMosaicML — APPROVED** [review]
LGTM! Confirmed this fixes things

**Comment by karan6181** [comment]
@Landanjs wondering the reason behind pinning the old streaming version `0.2.1`? Is it because of some hang that you have observed earlier ?

**Review by bandish-shah — CHANGES_REQUESTED** [review]
Would it be possible to use the seed workaround and not downgrade streaming?

**Comment by dakinggg** [comment]
Yeah, landan just pushed that change, i think at the same time you commented.

**Review by bandish-shah — APPROVED** [review]
Beat me to it!  Thanks!

File: examples/resnet/main.py [hunk 1]
```diff
@@ -33,6 +33,7 @@ def build_logger(name: str, kwargs: Dict):
 
 
 def main(config):
+    reproducibility.seed_all(cfg.seed)
```
Comment (dakinggg) [inline examples/resnet/main.py:36]:
```suggestion
    reproducibility.seed_all(config.seed)
```

File: examples/resnet/requirements.txt [hunk 2]
```diff
@@ -1,2 +1,2 @@
-mosaicml[streaming,wandb]>=0.11.0,<0.13
+mosaicml[wandb]>=0.11.0,<0.13
```
Comment (dakinggg) [inline examples/resnet/requirements.txt:1]:
```suggestion
mosaicml[streaming,wandb]>=0.11.0,<0.13
```

**Pull request merged**